### PR TITLE
[r] Support callers not having done `library(tiledb)`

### PR DESCRIPTION
**Pull request by johnkerl**
Working on a repro with @mlin I found that in a simple script that _doesn't_ do `library(tiledb)` -- only `library(tiledbsoma)` -- we get

```
Error in config(tiledb::tiledb_ctx()) : could not find function "config"
```

Repro script:

```
#!/usr/bin/env Rscript

library(tiledbsoma)

exp <- SOMAExperiment$new('path/to/data')
sdf <- exp$ms$get("RNA")$var
sdf$read(iterated=TRUE)
print(sdf$read_next())
```

### CODE REVIEW

File: apis/r/man/tiledb_embedded_version.Rd [hunk 1]
```diff
@@ -0,0 +1,11 @@
+% Generated by roxygen2: do not edit by hand
```
Comment (johnkerl) [inline apis/r/man/tiledb_embedded_version.Rd:1]: missed on previous PR #1077

**Review by eddelbuettel — APPROVED** [review]
Good catch -- and :shipit:

**Comment by eddelbuettel** [comment]
That's the cost of _not_ running examples :face_with_head_bandage:

File: apis/r/tests/testthat/test_SOMAReader-Iterated.R [hunk 2]
```diff
@@ -15,7 +15,7 @@ test_that("Iterated Interface from SOMAReader", {
     expect_true(dir.exists(uri))
 
     ctx <- tiledb_ctx()
-    sr <- sr_setup(uri, config=as.character(config(ctx)))
+    sr <- sr_setup(uri, config=as.character(tiledb::config(ctx)))
```
Comment (eddelbuettel) [inline apis/r/tests/testthat/test_SOMAReader-Iterated.R:18]: Or call `library(tiledb)` ?  It is a dependency of the package...

Comment (eddelbuettel) [reply apis/r/tests/testthat/test_SOMAReader-Iterated.R:18]: Witness:

```sh
edd@rob:~/git/tiledb-soma/apis/r/tests/testthat(main)$ grep library *
test_SOMAReader-Arrow.R:    library(arrow)
test_SOMAReader-Arrow.R:    library(tiledb)
test_SOMAReader-Iterated.R:    library(arrow)
test_SOMAReader-Iterated.R:    library(bit64)
test_SOMAReader-Iterated.R:    library(data.table)
test_SOMAReader-Iterated.R:    library(dplyr)
test_SOMAReader-Iterated.R:    library(tiledb)
edd@rob:~/git/tiledb-soma/apis/r/tests/testthat(main)$ 
```

Comment (eddelbuettel) [reply apis/r/tests/testthat/test_SOMAReader-Iterated.R:18]: So given this line you really do not need this:

https://github.com/single-cell-data/TileDB-SOMA/blob/f9e29883c7de5e4c93b3b0a68a3fe69fcceee362/apis/r/tests/testthat/test_SOMAReader-Iterated.R#L8

Comment (johnkerl) [reply apis/r/tests/testthat/test_SOMAReader-Iterated.R:18]: https://github.com/single-cell-data/TileDB-SOMA/pull/1084

**Pull request merged**